core/src/main/java/hudson/WebAppMain.java

```diff
@@ -130,7 +130,7 @@ public Locale get() {
             final FileAndDescription describedHomeDir = getHomeDir(event);
             home = describedHomeDir.file.getAbsoluteFile();
             home.mkdirs();
-            LOGGER.info("Jenkins home directory: "+ home +" found at: " + describedHomeDir.description);
+            LOGGER.finest("Jenkins home directory: "+ home +" found at: " + describedHomeDir.description);
```

Comment:
This an important debugging help and shouldn't be lower than the default visible INFO.